__tests__/app.test.js

```diff
@@ -72,36 +72,83 @@ describe('GET /api/articles/:article_id', () => {
             .expect(200)
             .then(({ body }) => {
                 expect(body.article).toStrictEqual(
-                    expect.objectContaining({
-                        author: expect.any(String),
-                        title: expect.any(String),
-                        article_id: expect.any(Number),
-                        body: expect.any(String),
-                        topic: expect.any(String),
-                        created_at: expect.any(String),
-                        votes: expect.any(Number)
-                    })
+                    {
+                        article_id: 3,
+                        title: "Eight pug gifs that remind me of mitch",
+                        topic: "mitch",
+                        author: "icellusedkars",
+                        body: "some gifs",
+                        created_at: "2020-11-03T09:12:00.000Z",
+                        votes: 0,
+                      }
                 )
             })
     });
-    it('should return a status 404 and an error message if passed a non-existent ID', () => {
+});
+describe('GET /api/articles/:article_id/comments', () => {
+    it('should return a status 200 and an array of comment objects', () => {
         return request(app)
-            .get("/api/articles/4321")
-            .expect(404)
+            .get("/api/articles/9/comments")
+            .expect(200)
             .then(({ body }) => {
-                expect(body).toStrictEqual({
-                    Message: "Article ID not found."
-                })
+                expect(body.comments).toHaveLength(2)
+                body.comments.forEach((comment) => {
+                    expect(comment).toStrictEqual(
+                        expect.objectContaining({
+                            comment_id: expect.any(Number),
+                            votes: expect.any(Number),
+                            created_at: expect.any(String),
+                            author: expect.any(String),
+                            body: expect.any(String),
+                        })
+                        )
+                    })
             })
     });
-    it('should return a status 400 and an error message if passed an invalid format ID', () => {
+    it('should sort the response by the most recent comment first (descending order)', () => {
         return request(app)
+            .get("/api/articles/9/comments")

```

Comment:
We don't need the extra .get here or the extra .expect(200) as that is handled in the above test. You could simply add an expect line for the sorted test to the above test block